WordPress/Classes/ViewRelated/Me/AccountSettingsViewController.swift

```diff
@@ -30,39 +29,47 @@ private struct AccountSettingsController: SettingsController {
     // MARK: - Initialization
 
     let service: AccountSettingsService
+    var settings: AccountSettings? {
+        didSet {
+            NSNotificationCenter.defaultCenter().postNotificationName(ImmuTableViewController.controllerChangedNotification, object: nil)
```

Comment:
I think this is a side effect of just trying to replace RxSwift without rethinking the design much. What we're trying to notify is that the _controller_ (ImmuTableController) has changed. since `ImmuTableController` is a protocol, I couldn't namespace the notification constant inside it, so I added it to `ImmuTableViewController`.

That said, I think `modelChangedNotification` might make more sense, as the controller hasn't necessarily _changed_, it'll just respond differently to its methods. In this implementation it actually changes since we're caching values for `settings` and `noticeMessage`, but those could have been computed.

On a more general note, `Controller` starts to feel like another failed experiment, and we might be better reframing it as a `ViewModel` to avoid confusion.